testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1965,10 +1974,10 @@ public void testUpdateAfterSchemaEvolution()
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
-    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS, ICEBERG_REST})
     public void testUpdateOnPartitionColumn()
     {
-        String baseTableName = "test_update_on_partition_column" + randomTableSuffix();
+        String baseTableName = toLowerCase("test_update_on_partition_column" + randomTableSuffix());
```

Comment:
Why `toLowerCase`? i think the `baseTableName` is already in lowercase, at least that was the intention here

Reply:
this is just in case somebody updates the table name to have mixed case. It seems easier to align all tables names to be lower case

Reply:
> It seems easier to align all tables names to be lower case to prevent mixed case issues

agreed

> this is just in case

this is unrelated change in a big PR.
i'd recommend backing it out, if this is "just in case".

Reply:
Removed references to `toLowerCase`